ArtifactOfOrder/ArtifactOfOrder.cs

```diff
@@ -3,23 +3,35 @@
 using R2API.Utils;
 using RoR2;
 using UnityEngine;
+using System.Reflection;
+using R2API;
+
 
 namespace ArtifactOfOrder
 {
     [NetworkCompatibility(CompatibilityLevel.EveryoneMustHaveMod, VersionStrictness.EveryoneNeedSameModVersion)]
     [BepInDependency("com.bepis.r2api")]
-    [BepInPlugin("dev.tsunami.ArtifactOfOrder", "ArtifactOfOrder", "1.0.1")]
+    [BepInPlugin("dev.tsunami.ArtifactOfOrder", "ArtifactOfOrder", "1.0.2")]
+    [R2APISubmoduleDependency(nameof(ArtifactAPI))]
+
     public class ArtifactOfOrder : BaseUnityPlugin
     {
+
+        // Artifact definition
+        public ArtifactDef Order = ScriptableObject.CreateInstance<ArtifactDef>();
         public void Awake()
         {
             Order.nameToken = "Artifact of Order";
             Order.descriptionToken = "Applies the sequencing effect of a Shrine of Order to a player upon death.";
-            Order.smallIconDeselectedSprite = CreateSprite(Properties.Resources.artifact_unselected);
-            Order.smallIconSelectedSprite = CreateSprite(Properties.Resources.artifact_selected);
+            using (var stream = Assembly.GetExecutingAssembly().GetManifestResourceStream("ArtifactOfOrder.artifactoforder"))
+            {
+                var bundle = AssetBundle.LoadFromStream(stream);
+                Order.smallIconSelectedSprite = bundle.LoadAsset<Sprite>("Assets/Import/Icons/artifact_selected.png");
+                Order.smallIconDeselectedSprite = bundle.LoadAsset<Sprite>("Assets/Import/Icons/artifact_unselected.png");
```

Comment:
Okay, I'm fine with switching to an AssetBundle if it simplifies some code.

Reply:
The change to an AssetBundle makes the whole Resources.Designer.cs Class obsolete. So i think the code is much easier to understand. 
But the main reason for the change was that i wasn't able to fix it without it.